### PR TITLE
Added how to run test app

### PR DESCRIPTION
**Pull request by matthieubosquet**
Added a short snippet to the README describing how to use the test app.

- [x] All acceptance criteria are met.
- [x] Relevant documentation, if any, has been written/updated.
- [x] The changelog has been updated, if applicable.
- [x] New functions/types have been exported in `index.ts`, if applicable.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,3 +60,15 @@ The `solid-client-authn` libraries are compatible with [NSS](https://github.com/
 
 - [Using Inrupt Solid JavaScript Client Libraries to authenticate](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/authenticate/)
 - [Inrupt documentation Homepage](https://docs.inrupt.com/)
+
+## How to run the test client login app?
+
+```javascript
```
Comment (RubenVerborgh) [inline README.md:66]: Not JavaScript though 😉

File: README.md [hunk 2]
```diff
@@ -60,3 +60,15 @@ The `solid-client-authn` libraries are compatible with [NSS](https://github.com/
...
+git clone https://github.com/inrupt/solid-client-authn-js
+cd solid-client-authn-js
+npm i
```
Comment (pmcb55) [inline README.md:69]: Not sure if you'd want `npm i` here or `npm ci` - @NSeydoux ...?

Comment (NSeydoux) [reply README.md:69]: I think you'd prefer `npm ci`, even though I'll make sure dependabot updates the dependencies if the demo app as well. Also, generally, we tend to use full names for the commands (e.g. `npm install`) in tutorials :)

File: README.md (hunk 1 above)
Comment (pmcb55) [reply README.md:66]: Yep, should be ```shell

Comment (matthieubosquet) [reply README.md]: Oops I used bash, but you're right, shell is more generic. I didn't know it was available.

**Review by NSeydoux — APPROVED** [review]

**Pull request merged**